### PR TITLE
Project documentation and additional examples.

### PR DESCRIPTION
**Pull request by jagoda**
This change documents the functionality of the vision plugin and
provides additional examples for using vision with several major
templating engines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,396 @@ Templates rendering plugin support for hapi.js.
 [![Build Status](https://secure.travis-ci.org/hapijs/vision.png)](http://travis-ci.org/hapijs/vision)
 
 Lead Maintainer - [Eran Hammer](https://github.com/hueniverse)
+
+**vision** decorates the [server](https://github.com/hapijs/hapi/blob/master/API.md#server) and
+[reply](https://github.com/hapijs/hapi/blob/master/API.md#reply-interface) interfaces with additional
+methods for managing view engines that can be used to render templated responses. **vision** also
+provides a built-in [handler](https://github.com/hapijs/hapi/blob/master/API.md#serverhandlername-method)
+implementation for creating templated responses.
+
```
Comment (hueniverse) [inline README.md]: You should add an example here for registering the plugin since hapi 9 will remove it from the built-in support.

**Comment by hueniverse** [comment]
@jagoda did you want to take over this module?

File: README.md (hunk 1 above)
Comment (jagoda) [inline README.md]: Sounds good. I will address the registration example as a separate change. I borrowed heavily from the hapi API.md when putting this together and tried to verify that everything was covered. I'm happy to rework things if I missed something though.

**Comment by jagoda** [comment]
Ya, that would be great!

**Comment by hueniverse** [comment]
@jagoda you are all set. Make sure to follow all the policies listed in hapijs/contrib and ask for help if you are not sure. I would recommend getting someone from the core team to review PR you make yourself before merging.

**Comment by jagoda** [comment]
Awesome! Will do.

**Pull request merged**